### PR TITLE
[CI] Trigger bootstrap bundle creation after Habitat release

### PR DESCRIPTION
**Pull request by christophermaier**
This adds a new Buildkite pipeline for post-Habitat release tasks,
which right now consists of just creating the bootstrap bundle.

The old code was removed, in favor of a more streamlined `hab pkg
download`-based approach.

Signed-off-by: Christopher Maier <cmaier@chef.io>

### CODE REVIEW

File: .expeditor/scripts/generate_bootstrap_bundle.sh [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# Conveniently, we have just downloaded a hart file for that. To make
+# things easy on us, we can extract the stand-alone `hab` binary from
+# that hart file (this assumes Linux packages, naturally) and store it
+# at `bin/hab` in the tar archive.
+
+# The regex is to add the most general placeholders for "version" and
+# "release" in the hart name, while also avoiding similarly-named
+# packages like
+# `core-hab-launcher-${VERSION}-${RELEASE}-x86_64-linux.hart`, etc.
+hab_hart=$(find "${sandbox_dir}"/artifacts -type f -regex '.*/core-hab-[^-]*-[^-]*-x86_64-linux.hart')
```
Comment (smacfarlane) [inline .expeditor/scripts/generate_bootstrap_bundle.sh]: Does this run after builder-worker is built and promoted, and if not, will this always return a single hart that is the latest? 

What I'm thinking here is that if builder-worker hasn't been built and promoted before this runs, we could end up with two versions of `core/hab` in the artifacts path.  `habitat/builder-worker` deps on `core/hab-studio` which deps on `core/hab-backline` which deps on `core/hab-plan-build` which deps on `core/hab`.

File: .expeditor/builder_seed.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Supervisor and prerequisites
+core/hab-launcher
+core/hab
```
Comment (smacfarlane) [inline .expeditor/builder_seed.txt]: should `core/hab-plan-build` and `core/hab-backline` also be included here?  They'll get pulled in as deps, but do we want to be explicit?  

Also, do we want to include the docker exporter for Builder?

Comment (christophermaier) [reply .expeditor/builder_seed.txt]: All those get pulled in as dependencies... I just wanted to keep things simple and high-level.

File: .expeditor/scripts/generate_bootstrap_bundle.sh (hunk 1 above)
Comment (christophermaier) [reply .expeditor/scripts/generate_bootstrap_bundle.sh]: Ugh, that is a very good catch :man_facepalming: 

Once we get the builder-worker incorporated into the pipeline (https://github.com/habitat-sh/builder/issues/1217), it won't be an issue. In the meantime, we can probably just grab the Habitat `manifest.json` file to get a concrete `hab` version. Alternatively, we could sort the results of this `find` and grab the "biggest" one.

I'll play around and see what looks best.

Comment (christophermaier) [reply .expeditor/scripts/generate_bootstrap_bundle.sh]: Addressed in https://github.com/habitat-sh/builder/pull/1245/commits/b55f30367d193e3ba1e6c2ef4e17b5c9145b4303

**Pull request merged**